app/models/journeys/Journey.scala

```diff
@@ -100,6 +109,10 @@ object Journey extends Enum[Journey] with utils.PlayJsonEnum[Journey] {
     override val pageKeys: List[PageName] = List(
       ExpensesCategoriesPage.pageName
     )
+    override def startUrl(taxYear: TaxYear, businessId: BusinessId, mode: Mode): String =
+      expenses.tailoring.routes.ExpensesCategoriesController.onPageLoad(taxYear, businessId, mode).url
+
+    override def startPage: QuestionPage[_] = ExpensesCategoriesPage
   }
```

Comment:
we'll need to come up with a solution for the WorkplaceRunningCosts journey were there are two potential start pages

Reply:
good point. I will have a look

Reply:
will get back to it later, but I allow now to pass start page in the HopChecker